test/new-e2e/tests/containers/k8s_test.go

```diff
@@ -842,7 +843,50 @@ func (suite *k8sSuite) testAdmissionControllerPod(namespace string, name string,
 }
 
 func (suite *k8sSuite) TestContainerImage() {
-	suite.EventuallyWithTf(func(c *assert.CollectT) {
+	sendEvent := func(alertType, text string) {
+		if _, err := suite.datadogClient.PostEvent(&datadog.Event{
+			Title: pointer.Ptr(suite.T().Name()),
+			Text: pointer.Ptr(fmt.Sprintf(`%%%%%%
+`+"```"+`
+%s
+`+"```"+`
+ %%%%%%`, text)),
+			AlertType: &alertType,
+			Tags: []string{
+				"app:agent-new-e2e-tests-containers",
+				"cluster_name:" + suite.clusterName,
+				"contimage:ghcr.io/datadog/apps-nginx-server",
```

Comment:
Don't we already have the `image_name` tag for that ?

Reply:
I wanted to have a dedicated tag to distinguish events generated by the different tests.